### PR TITLE
[LTC] Implement Elrond of the White Council

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Elrond of the White Council]]

File: Mage.Sets/src/mage/cards/e/ElrondOfTheWhiteCouncil.java [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                true
+        );
+        vote.doVotes(source, game);
+
+
+        Set<Permanent> chosenCreatures = new HashSet<>();
+
+        // For each fellowship vote,
+        for (UUID playerId : game.getState().getPlayersInRange(source.getControllerId(), game)) {
+            int countFellowship = (int) vote.getVotes(playerId).stream().filter(b -> b).count();
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/e/ElrondOfTheWhiteCouncil.java:100]: Curious- why are we filtering from b to b, and why is the cast needed?

Comment (Susucre) [reply Mage.Sets/src/mage/cards/e/ElrondOfTheWhiteCouncil.java:100]: count() returns a long.
The filter is that the votes are true for fellowship, and false for aid, and i wanted only the fellowship ones.

Comment (xenohedron) [reply Mage.Sets/src/mage/cards/e/ElrondOfTheWhiteCouncil.java:100]: Thanks for explaining, I get it now.

**Review by xenohedron — APPROVED** [review]

**Pull request merged**